src/content/docs/journey-app-slow/root-causes.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: Prepare to triage your application
+metaDescription: 
+---
+
+import journeyWebTrans from 'images/journey_screenshot-web-transactions.webp'
+
+Your app is slow. Now what? How do you track down the root cause of your app's slowness?
+
+This tutorial walks you through how to use New Relic to triage your application and identify the root cause of your app's degraded performance. You'll start by "instrumenting" your app with an agent, which means installing a piece of code that reports data about your app to New Relic. Using this data in New Relic, you'll dig into the performance of your application and identify slow transactions, slow database queries, or slow external services as your issue - or possibly all three!
```

Comment:
I like this, it's punchy and human! 

``` issue - or possibly all three!```

^ Nitpick: If you use an em-dash (and while we should be careful about them, I'm okay with them in docs like this), it should be the true em-dash character and it should not have spaces around it. Instructions here: https://docs.newrelic.com/docs/style-guide/word-choice/usage-dictionary/#em-dash